terratorch/cli_tools.py

```diff
@@ -101,6 +99,20 @@ def save_prediction(prediction, input_file_name, out_dir, dtype:str="int16"):
     logging.info(f"Saving output to {out_file_name} ...")
     write_tiff(result, os.path.join(out_dir, out_file_name), metadata)
 
+def import_custom_modules(custom_modules_path):
+    if custom_modules_path:
+        if os.path.isdir(custom_modules_path):
+
+            # Add 'custom_modules' folder to sys.path
+            workdir, module_dir = os.path.split(custom_modules_path)
+
+            if not workdir:
+                workdir = "."
+
+            sys.path.append(workdir)
+            logging.info(f"Found {custom_modules_path}")
+
+            importlib.import_module(module_dir)
```

Comment:
I think you need a try-except here because what if you try to import the custom module, but it fails? If you catch the exception, you can probably give the user a more helpful error than whatever importlib does.